### ISSUE TITLE
As user I want see description for map view button

### ISSUE THREAD

**Issue by teittla** [HSLdevcom/jore4#1125]
Ilmestymään hiiren päälle symbolin viedessä selite "Näytä kartalla" sille 

![Image](https://user-images.githubusercontent.com/105207476/214776011-8fa29392-e06d-487f-98d3-13edbc368c1b.png)

**Comment by juhakolari** [comment]
Have not tested all buttons myself, but all icons and buttons should have alt text. This will also come up in accessibility testing.
